validator/src/test/java/org/odftoolkit/odfvalidator/InvalidPackageTest.java

```diff
@@ -157,10 +157,6 @@ public void validateEncryptedODT() {
       t.printStackTrace(new PrintWriter(errors));
       Assert.fail(t.toString() + "\n" + errors.toString());
     }
-    Assert.assertTrue(output.contains("Fatal: only DEFLATED entries can have EXT descriptor"));
-    Assert.assertTrue(
-        output.contains(
-            "Fatal: The document is encrypted. Validation of encrypted documents is not supported."));
```

Comment:
we can only validate the manifest.xml in encrypted files; i think this should be a warning, maybe a fatal error is a bit too much.

Reply:
The got removed in 60993ebd75e5cf54e643e097d because this does not happen anymore

Reply:
hmm i think there should be some output that the document streams weren't actually validated though.